### ISSUE TITLE
Lazy set up example

### ISSUE THREAD

**Issue by liamwh** [Isti115/agda.nvim#10]
More and more Neovim users are converging on the [lazy.nvim](https://github.com/folke/lazy.nvim) package manager by folke. I feel it would help others get onboard with this plugin if there a known good configuration of adga.nvim using lazy.nvim was added to the readme. Is anyone open to adding this?

**Comment by isti115** [comment]
@liamwh Good idea, I have also switched to `lazy` about a year ago. I'll add a minimal config to the README soon.
